### PR TITLE
Add config list subcommand

### PR DESCRIPTION
**Pull request by jimmidyson**

### CODE REVIEW

**Comment by dlorenc** [comment]
Thanks!

**Comment by jimmidyson** [comment]
Just wondering if it makes more sense to be `config show` rather than `config list`?

**Comment by r2d4** [comment]
If anything, I think we should adhere to the convention `kubectl config view`.  But I'm ok with `config list` too.

**Comment by jimmidyson** [comment]
Let me switch to `config view` - like the kubectl consistency you mentioned.

**Comment by jimmidyson** [comment]
Done -changed to `config view`

File: pkg/minikube/constants/constants.go [hunk 1]
```diff
@@ -65,7 +65,8 @@ const (
 	DefaultVMDriver     = "virtualbox"
 	DefaultStatusFormat = "minikubeVM: {{.MinikubeStatus}}\n" +
 		"localkube: {{.LocalkubeStatus}}\n"
-	DefaultListFormat         = "- {{.AddonName}}: {{.AddonStatus}}\n"
+	DefaultAddonListFormat    = "- {{.AddonName}}: {{.AddonStatus}}\n"
+	DefaultConfigViewFormat   = "- {{.ConfigName}}: {{.ConfigValue}}\n"
```
Comment (r2d4) [inline pkg/minikube/constants/constants.go]: This should be {{.ConfigKey}} {{.ConfigValue}} as you have it above

- E1014 11:34:27.047476  120837 config_view.go:72] Error executing view template: template: view:1:4: executing "view" at <.ConfigName>: can't evaluate field ConfigName in type config.ConfigViewTemplate

https://github.com/kubernetes/minikube/pull/695/files#diff-42345276571c35ff317855ef4335db2aR34

**Pull request merged**